### PR TITLE
FD.io users: update url to point to IPng Networks articles

### PR DESCRIPTION
**Pull request by dwallacelf**
- instead of linking to top site, link to the blog articles about VPP & other networking topics

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -240,7 +240,7 @@ <h2 class="pb-4">FD.io Users</h2>
 					)
 				}}
 				{{ partial "usecases.html" (dict
-					"href" "https://ipng.ch"
+					"href" "https://ipng.ch/s/articles/"
 					"img" "/img/logos/IPng-network.png"
 					"title" "IPng Networks"
 					"content" "IPng Networks is a contributer to the Linux CP plugin and is also a downstream user"
```
Comment (pimvanpelt) [inline layouts/index.html:246]: You could s/downstream user/source of in-depth technical articles on the VPP ecosystem/
I don't think anybody cares that I use VPP. What they care about perhaps more, is that they can turn to the site for insights and knowledge.

**Review by pimvanpelt — APPROVED** [review]
Looks good. One nit on 'downstream user' versus 'source of knowledge'. PTAL

File: layouts/index.html (hunk 1 above)
Comment (pimvanpelt) [reply layouts/index.html:246]: by the way, looking at the PR topic - it's IPng (which is an original term from Steve Deering from the IAB when describing IPv6), not IPNg. Full name IPng Networks, and that might explain where `lcpng` comes from.

Comment (dwallacelf) [reply layouts/index.html]: Updated with suggested changes.  Will merge once you have reviewed/approved the updates.

ps. Thanks for the history lesson :)

**Review by pimvanpelt — APPROVED** [review]
Excellent and thank you

**Pull request merged**